src/main/kotlin/com/terning/server/kotlin/application/scrap/dto/DetailedMonthlyScrapResponse.kt

```diff
@@ -0,0 +1,65 @@
+package com.terning.server.kotlin.application.scrap.dto
+
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+data class DetailedMonthlyScrapResponse(
+    val dailyGroups: List<DetailedScrapGroup>,
+)
+
+data class DetailedScrapGroup(
+    val deadline: String,
+    val scraps: List<DetailedScrap>,
+)
+
+data class DetailedScrap(
+    val announcementId: Long,
+    val companyImageUrl: String,
+    val title: String,
+    val workingPeriod: String,
+    val isScrapped: Boolean,
+    val hexColor: String,
+    val deadline: LocalDate,
+    val startYear: Int,
+    val startMonth: Int,
+    val formattedDeadline: String,
+) {
+    val deadlineText: String = "${deadline.year}년 ${deadline.monthValue}월 ${deadline.dayOfMonth}일"
+    val startYearMonth: String = "${startYear}년 ${startMonth}월"
+
+    companion object {
+        fun from(
+            announcementId: Long,
+            companyImageUrl: String,
+            title: String,
+            workingPeriod: String,
+            isScrapped: Boolean,
+            hexColor: String,
+            deadline: LocalDate,
+            startYear: Int,
+            startMonth: Int,
+            clock: java.time.Clock,
```

Comment:
`java.time`은 임포트로 빼주는 거 어떤가용?
```suggestion
            clock: Clock,
```

Reply:
아이코 제가 그 부분을 놓쳤네요..! 바로 수정하겠습니다. 감사합니다! 😊